<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAdminClass</class>
 <widget class="QWidget" name="QtAdminClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAdminClass</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>631</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="unaprovedCredits">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UNAPROVED CREDITS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CREDITS LIST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activeCreditsButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACTIVE CREDITS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Desktop/icons8-synchronize-26.png</normaloff>../../../../Desktop/icons8-synchronize-26.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
